life/modernized/27-chapter-27.adoc

```diff
@@ -5054,18 +5265,25 @@ nor distance wear out of our remembrances,
 and in which we desire to be remembered of you to the Lord of our household,
 we dearly and tenderly salute and embrace you, and remain,
 
+[.signed-section-closing]
 Your loving and faithful brethren,
 
+[.signed-section-signature]
 William Penn, John Everot, Thomas Story.
 
-Postscript.
+[.postscript]
```

Comment:
So you can't put a [.postscript] tag over a blank line. Jared is actually working on re-doing the postscript syntax right now, but for now, lets kill the word Postscript in the blurb, and do this section like I show below, and we'll explain the new syntax later (and it will shortly be in the style guide)

```
[.postscript]
P. S. Friends here have been ...etc....

```